RELEASENOTES.md

```diff
@@ -1,5 +1,120 @@
+- While the install in running, a status window will display.
+
+•	Once installation is complete:
+- ApplicationBrowser will be installed under C:\Program Files (x86)\Application Browser\
+- Shortcuts should now be available on the windows start menu. You may also want to right-click this icon in the application bar to pin it.
+- You will need to configure the application environments. The application will prompt the user if the configuration file is missing or is invalid.
+
+### Using Application Browse exe artifact
+While the install in running, a loading icon will display.
+
+NOTE: The application will open as part of the install process. It may automatically restart once before it finishes. 
+
+•	In the event of an error, you may not have write permission set on the install directory. 
+- Go to C:\Users\<username>\AppData\Local
+- If the ApplicationBrowser folder does not exist, create it.
+- Right-click the new ApplicationBrowser folder, click on permissions and deselect the Read-only checkbox.
+ -	Apply the changes and rerun the install.
+
+•	Now that the install is complete:
+- Shortcuts should now be available on the desktop. You may also want to right-click this icon in the application bar to pin it.
+- Unless the application has been preconfigured, you will need to configure the application environments. The application will prompt the user if the configuration file is missing or is invalid.
+
+## How to Configure
+After the client is installed, you may get a prompt that config.json file is missing.
+* If ApplicationBrowser-<version>.msi was used to install Application Browser, place the config.json file under C:\Program Files (x86)\Application Browser
+* If ApplicationBrowser-<version>-setup.exe was used to install Application Browser, place the config.json file under C:\Users\<username>\AppData\Local\ApplicationBrowser\app-<version>
+Create this file and populate it with environment URLs. Here is a sample configuration file.
+```
+{
+  "flash":
+  {
+    "path":"C:\\Windows\\System32\\Macromed\\Flash\\pepflashplayer64_28_0_0_137.dll"
+  },
+
+ "environments":
+  [
+    {
+      "id": "test",
+      "url": "https://www.google.it",
+      "label": "Test"
+    }, {
+      "id": "sandbox",
+      "url": "https://us.yahoo.com/",
+      "label": "Sandbox"
+    }, {
+      "id": "production",
+      "url": "https://www.facebook.com",
+      "label": "Production"
+    }
+  ],
+
+  "whenUrl": [
+    {
+      "endsWith": "PPSS/PPSS",
+      "thenAppend": "?control=loginPage"
+    }
+  ]
+}
+```
+
+* flash > path (optional) - The path to the installed pepper flashplayer.dll file. The application will attempt to use this path to the Pepper Flash Player. If not found, the application will attempt to find an installed version.
+* environments (required) - An array of environments that will be available on the initial page. Each environment has the following properties:
+* id (required) – A unique id for the environment.
```

Comment:
id, url, and label should be indented under environments.
![image](https://user-images.githubusercontent.com/73573586/98389675-fe70ab00-2019-11eb-8bc0-310c75f4ba32.png)